tests/integration/cloud/providers/profitbricks.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+'''
+    :codeauthor: :email:`Ethan Devenport <ethand@stackpointcloud.com>`
+'''
+
+# Import Python Libs
+from __future__ import absolute_import
+import os
+import random
+import string
+
+# Import Salt Testing Libs
+from salttesting import skipIf
+from salttesting.helpers import ensure_in_syspath, expensiveTest
+
+ensure_in_syspath('../../../')
+
+# Import Salt Libs
+import integration
+from salt.config import cloud_providers_config
+from salt.ext.six.moves import range
+
+# Import Third-Party Libs
+try:
+    import libcloud # pylint: disable=unused-import
+    HAS_LIBCLOUD = True
+except ImportError:
+    HAS_LIBCLOUD = False
+
+def __random_name(size=6):
+    '''
+    Generates a random cloud instance name
+    '''
+    return 'CLOUD-TEST-' + ''.join(
+        random.choice(string.ascii_uppercase + string.digits)
+        for x in range(size)
+    )
+
+# Create the cloud instance name to be used throughout the tests
+INSTANCE_NAME = __random_name()
+PROVIDER_NAME = 'profitbricks'
+DRIVER_NAME = 'profitbricks'
+
+@skipIf(HAS_LIBCLOUD is False, 'salt-cloud requires >= libcloud 0.13.2')
```

Comment:
This `skipIf` should be changed to look for `HAS_PROFITBLOCKS` and the a message saying that profitblocks version x is required.